src/DAL/migration/fullSchema.sql

```diff
@@ -4,25 +4,25 @@
 
 CREATE TABLE IF NOT EXISTS "polygon_parts".parts
 (
-    "partId" integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
-    "recordId" uuid NOT NULL,
+    "part_id" integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    "record_id" uuid NOT NULL,
     "id" text COLLATE pg_catalog."default",
     "name" text COLLATE pg_catalog."default",
-    "updatedInVersion" text COLLATE pg_catalog."default",
-    "ingestionDateUTC" timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    "imagingTimeBeginUTC" timestamp with time zone NOT NULL,
-    "imagingTimeEndUTC" timestamp with time zone NOT NULL,
-    "resolutionDegree" numeric NOT NULL,
-    "resolutionMeter" numeric NOT NULL,
-    "sourceResolutionMeter" numeric NOT NULL,
-    "horizontalAccuracyCE90" real,
+    "updated_in_version" text COLLATE pg_catalog."default",
+    "ingestion_date_UTC" timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
UTC is upper case, will it work in queries withaout marking "" as it works with lower case by default?

Reply:
according to docs, upper cases on postgres are sensitive, its better to add "" just for safe

Reply:
done